### ISSUE TITLE
Unexpected results when using map-keys-quotes and Sass operators

### ISSUE THREAD

**Issue by tysongach** [kristerkari/stylelint-scss#386]
The [map-keys-quotes](https://github.com/kristerkari/stylelint-scss/tree/master/src/rules/map-keys-quotes) linter is throwing false positives when using [Sass operators](https://sass-lang.com/documentation/operators) (such as `*` or `+`) inside values.

Given the following map, map-keys-quotes throws an error:

```scss
$map: (
  "key-one": 0,
  "key-two": $variable * 2,
);
```

Upon investigating, it appears to want the value `$variable * 2` to be quoted. In my testing, if I quote that value, the linter no longer throws an error:

```diff
-  "key-two": $variable * 2,
+  "key-two": "$variable * 2",
```

My expectation is that operations on variables would not throw an error.

**Comment by kristerkari** [comment]
Thanks! That definitely should not give a warning.
